<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CodeEditor" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1387</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionHistory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionLineWrap"/>
    <addaction name="actionShowLineNumber"/>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionFontBackgroundColor"/>
    <addaction name="actionEditorBackgroundColor"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionShowToolBar"/>
    <addaction name="actionShowStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionHistory"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionShowLineNumber"/>
   <addaction name="actionLineWrap"/>
   <addaction name="separator"/>
   <addaction name="actionShowToolBar"/>
   <addaction name="actionShowStatusBar"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/new-file.png</normaloff>:/img/new-file.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/folder.png</normaloff>:/img/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/save-as.png</normaloff>:/img/save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/undo.png</normaloff>:/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/cut.png</normaloff>:/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/copy.png</normaloff>:/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/paste.png</normaloff>:/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/find-file.png</normaloff>:/img/find-file.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/replace.png</normaloff>:/img/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/font.png</normaloff>:/img/font.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
  </action>
  <action name="actionLineWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/linewrap.png</normaloff>:/img/linewrap.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionShowToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/toolbar.png</normaloff>:/img/toolbar.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/status.png</normaloff>:/img/status.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/redo.png</normaloff>:/img/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;)</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+Z</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionShowLineNumber">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/showlinenumber.png</normaloff>:/img/showlinenumber.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>显示行号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>设置字体颜色</string>
   </property>
  </action>
  <action name="actionEditorBackgroundColor">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>设置编辑器背景色</string>
   </property>
  </action>
  <action name="actionFontBackgroundColor">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>设置字体背景色</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/history.png</normaloff>:/img/history.png</iconset>
   </property>
   <property name="text">
    <string>最近历史</string>
   </property>
   <property name="toolTip">
    <string>最近历史</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
